Triton_Inference_Server_Python_API/run.sh

```diff
@@ -29,14 +29,14 @@ TAG=
 RUN_PREFIX=
 
 # Frameworks
-declare -A FRAMEWORKS=(["DIFFUSERS"]=1 ["TRT_LLM"]=2 ["IDENTITY"]=3)
```

Comment:
Nit: this file requires an updated copyright year. Out of scope I can't do a "suggested change"